Maple2.Server.Game/PacketHandlers/ItemUseHandler.cs

```diff
@@ -106,4 +117,91 @@ public class ItemUseHandler : PacketHandler<GameSession> {
             session.Player.Value.Unlock.Titles.Add(titleId);
         }
     }
+
+    private void HandleBuddyBadgeBox(GameSession session, IByteReader packet, Item item) {
+        string targetUser = packet.ReadUnicodeString();
+
+        if (targetUser == session.Player.Value.Character.Name) {
+            session.Send(NoticePacket.MessageBox(new InterfaceText(StringCode.s_couple_effect_error_openbox_myself_char)));
```

Comment:
Here and below, there is an implicit constructor for `InterfaceText` from `StringCode` so you can just pass `StringCode` in.